driver/conv_driver.hpp

```diff
@@ -439,6 +439,29 @@ int ConvDriver<Tgpu, Tref>::ParseCmdLineArgs(int argc, char* argv[])
 {
     inflags.Parse(argc, argv);
 
+    // try to set a default layout value for 3d conv if not specified from cmd line
+    int spatial_dim             = inflags.GetValueInt("spatial_dim");
+    bool layout_not_initialized = inflags.GetValueStr("in_layout").empty() &&
+                                  inflags.GetValueStr("fil_layout").empty() &&
+                                  inflags.GetValueStr("out_layout").empty();
```

Comment:
If only one layout is initialized, then the rest two will be left empty. Is it Ok? What about this:
```C++
const std::string default_layout = (spatial_dim == 2) ? "NCHW" : "NCDHW";
if(inflags.GetValueStr("in_layout").empty())
    inflags.SetValue("in_layout", default_layout);
/// the same for other two
```

Reply:
OK, I will change to this